modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/DDMServiceUpgrade.java

```diff
@@ -287,8 +289,10 @@ public void register(Registry registry) {
 		registry.register(
 			"3.2.3", "3.2.4", new UpgradeDDMContent(_jsonFactory));
 
+		registry.register("3.2.4", "3.2.5", new DummyUpgradeStep());
+
 		registry.register(
-			"3.2.4", "3.3.0",
+			"3.2.5", "3.3.0",
```

Comment:
@rodrigopaulino, why was it necessary?

Reply:
This is necessary every time an Upgrade needs to be backported. This reserves the version number that's going to be used on 7.2.x.

Reply:
Practically, if someone is trying to upgrade from 7.2 to the latest, Liferay will be able to pick it up from their schema version onwards.